### PR TITLE
Feature/add-reset-password-frontend

### PR DESCRIPTION
**Pull request by SerhiiPlotnikov**

### CODE REVIEW

**Comment by lenchvolodymyr** [comment]
you added a few commits but looks like you added a whole application. Did you do hard reset?

**Comment by SerhiiPlotnikov** [comment]
> you added a few commits but looks like you added a whole application. Did you do hard reset?

No, I have been writing my component, and then i decided to make pull request for a new state of develop branch. After that i made push request to my branch and got so much changed files

**Comment by TatsuUkraine** [comment]
@SerhiiPlotnikov please fix merge conflicts

File: frontend/src/components/auth/ResetPasswordForm.vue [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                                required
+                        />
+                    </VForm>
+                </VCardText>
+                <VCardActions>
+                    <VBtn @click="onResetPassword" class="reset-password-form-button mt-3" color="#3C57DE">
+                        Send Password Reset Link
+                    </VBtn>
+                </VCardActions>
+                <v-alert class="status-success" v-if="status==='success'" type="success">
```
Comment (TatsuUkraine) [inline frontend/src/components/auth/ResetPasswordForm.vue]: use PascalCase style for component naming, to keep app style consistent

File: frontend/src/components/auth/ResetPasswordForm.vue [hunk 2]
```diff
@@ -0,0 +1,167 @@
+<template>
+    <VContent>
+        <VFlex
+                lg6
```
Comment (TatsuUkraine) [inline frontend/src/components/auth/ResetPasswordForm.vue]: it seems you have unnecessary spaces here and in file in general

File: frontend/src/components/auth/ResetPasswordForm.vue [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+                                v-model="email"
+                                solo
+                                type="text"
+                                :rules="emailRules"
+                                required
+                        />
+                    </VForm>
+                </VCardText>
+                <VCardActions>
+                    <VBtn @click="onResetPassword" class="reset-password-form-button mt-3" color="#3C57DE">
```
Comment (TatsuUkraine) [inline frontend/src/components/auth/ResetPasswordForm.vue]: ensure that your code uses styleguide suggestions, where even binding should places last one

File: frontend/src/components/auth/ResetPasswordForm.vue (hunk 1 above)
Comment (TatsuUkraine) [inline frontend/src/components/auth/ResetPasswordForm.vue]: probably better avoid using v-if-else here. You can define computed prop like `showAlert` that will be true, when status not empty. Class value also can be move to computed prop and define it's value based on status, same for type

File: frontend/src/components/auth/ResetPasswordForm.vue [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+                resetPassword: RESET_PASSWORD
+            }),
+            onResetPassword() {
+                if (this.$refs.form.validate()) {
+                    this.resetPassword({
+                        email: this.email,
+                    }).then(response => {
+                        this.status = 'success';
+                        this.message = response;
+                    }, err => {
```
Comment (TatsuUkraine) [inline frontend/src/components/auth/ResetPasswordForm.vue]: better to use `catch` chain method

File: frontend/src/pages/ResetPassword.vue [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<template>
+    <ResetPasswordForm></ResetPasswordForm>
```
Comment (TatsuUkraine) [inline frontend/src/pages/ResetPassword.vue]: use self closing tags when it's possible

**Pull request merged**